xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/specifiers/LedgerIndexShortcut.java

```diff
@@ -0,0 +1,63 @@
+package org.xrpl.xrpl4j.model.client.specifiers;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import java.util.Objects;
+
+/**
+ * A String wrapping class that defines three static constants representing shortcut values
+ * which can be used to specify a ledger index in a rippled API request.
+ */
+public class LedgerIndexShortcut {
+
+  /**
+   * Request information about a rippled server's current working version of the ledger.
+   */
+  public static final LedgerIndexShortcut CURRENT = new LedgerIndexShortcut("current");
```

Comment:
Anyone know why codecov isn't picking these lines up as tested?

Reply:
Not sure; I've had it do things like that before too. Not sure if referencing it directly in a test would address it (though maybe you're already doing that).

Reply:
Showing as tested in the latest report.